coverage/base.css

```diff
@@ -0,0 +1,224 @@
+body, html {
```

Comment:
### Nitpick

- We don't need this commited to Github

Reply:
I resolved it